paas_app_charmer/django/charm.py

```diff
@@ -10,19 +8,14 @@
 import typing
 
 import ops
-
-# pydantic is causing this no-name-in-module problem
-from pydantic import BaseModel, Extra, Field, ValidationError  # pylint: disable=no-name-in-module
+from pydantic import BaseModel, Extra, Field, validator
 
 from paas_app_charmer._gunicorn.charm import GunicornBase
-from paas_app_charmer._gunicorn.charm_utils import block_if_invalid_config
-from paas_app_charmer.exceptions import CharmConfigInvalidError
-from paas_app_charmer.utils import build_validation_error_message
 
 logger = logging.getLogger(__name__)
 
 
-class DjangoConfig(BaseModel, extra=Extra.allow):  # pylint: disable=too-few-public-methods
+class DjangoConfig(BaseModel, extra=Extra.ignore):
```

Comment:
What's this change?

Reply:
As now the DjangoConfig is created with all the charm config, the attributes that are not in the DjangoConfig should just be ignored. Before, they were filtered before using the pydantic class.